modules/Messenger/messenger_manage_edit.php

```diff
@@ -147,7 +153,9 @@
 					$smsURL=getSettingByScope( $connection2, "Messenger", "smsURL" ) ;
 					$smsURLCredit=getSettingByScope( $connection2, "Messenger", "smsURLCredit" ) ;
 					if ($smsUsername == "" OR $smsPassword == "" OR $smsURL == "") {
-						$form->addRow()->addAlert(sprintf(__('SMS NOT CONFIGURED. Please contact %1$s for help.'), "<a href='mailto:" . $_SESSION[$guid]["organisationAdministratorEmail"] . "'>" . $_SESSION[$guid]["organisationAdministratorName"] . "</a>"), 'error');
+						$row = $form->addRow()->addClass('sms');
+							$row->addLabel('sms', __('SMS'))->description(__('Deliver this message to user\'s mobile phone?'));
```

Comment:
Is this label meant to be on the non-configured side of this conditional, or the configured side? I suspect maybe the latter 🤔

Reply:
Yes, you are right. Are you OK to tweak? Thanks!

Reply:
👍